Resources/Prototypes/_LostParadise/Entities/Mobs/Species/Tajaran/Body/Organs/Tajaran.yml

```
@@ -17,3 +24,9 @@
         reagents:
         - ReagentId: UncookedAnimalProteins
           Quantity: 5
+  - type: Metabolizer
+    maxReagents: 3
+    metabolizerTypes: [ Animal ]
```

yamllint

[error] too many spaces inside brackets

(brackets)